### PR TITLE
New DS6 notices/alerts style

### PR DESCRIPTION
**Pull request by ianmcburnie**
Issue #14

This PR consists of two parts:

1. updating current DS4 markup to also support non-inline SVG (i.e. allowing a span tag instead of svg tag for icon).
1. add DS6 styles for notice (only supports span tag for icon for now, not inline SVG)

screen shot 2018-02-22 at 1 38 57 pm

screen shot 2018-02-22 at 1 57 38 pm

### CODE REVIEW

**Comment by seangates** [comment]
@ianmcburnie Some margins look off, so give me a chance to review before merging. :grin:

File: src/less/notice/ds6/notice-base.less [hunk 1]
```diff
@@ -1,86 +1,121 @@
...
+.inline-notice__status span {
+    background-repeat: no-repeat;
+    background-size: contain;
+    display: inline-block;
+}
+
+.page-notice__status span {
     height: 28px;
-    margin: 0 4px;
+    margin: -4px 0 0 0;
```
[the remark this answers is not in the document]
Comment (ianmcburnie) [reply src/less/notice/ds6/notice-base.less]: Yeah the -4px is to line up with the text a little better. If I adjust the paddings maybe I can remove it.

The DSL seems to have some arbitrary amount of viewbox/framing around the SVG. I've tried to normalize that.

File: src/less/notice/ds6/notice-base.less [hunk 2]
```diff
@@ -1,86 +1,121 @@
...
-    margin-bottom: 8px;
-    h2,
-    h3,
-    h4 {
-        font-size: @ds6-font-size-20;
-        margin: 0;
-        width: 100%;
-    }
+    margin: 0;
+    padding: 16px;
```
[the remark this answers is not in the document]
Comment (ianmcburnie) [reply src/less/notice/ds6/notice-base.less]: Thanks. Will take a look.

File: src/less/notice/ds6/notice-base.less (hunk 1 above)
[the remark this answers is not in the document]
Comment (seangates) [reply src/less/notice/ds6/notice-base.less]: Yeah, I thought about this. I think the DSL has munched the image. (Which I took a screenshot of.)

I'm okay if you don't change this, just wanted to know the reasoning around it, and if I was seeing things incorrectly.

File: src/less/notice/ds6/notice-base.less (hunk 2 above)
[the remark this answers is not in the document]
Comment (ianmcburnie) [reply src/less/notice/ds6/notice-base.less]: You have good eyeballs. That left padding is noticeably off to me now you mention it!

Here it is `padding: 12px 16px 12px 12px`. Changing the top and bottom to 12px meant I could remove the `margin-top: -4px` and single line of text aligns nicely. Looks ok to me, even though the spec technically says 16px + 4px border for top and bottom:

screen shot 2018-02-23 at 12 29 03 pm

[the remark this answers is not in the document]
Comment (seangates) [reply src/less/notice/ds6/notice-base.less]: Much better! Thanks!!

[the remark this answers is not in the document]
Comment (seangates) [reply src/less/notice/ds6/notice-base.less]: Approved, pending your incoming commit. 😉

**Comment by JeffNixonCreative** [comment]
I would recommend not referencing the image assess on DSL for accurate specs. SVG’s have yet to be perfectly exported from Sketch or Illustrator without tweaking. I think the PDF should be a more accurate representation but if you have any questions or concerns I can help get them with the team.

**Comment by seangates** [comment]
@JeffNixonCreative Yeah, I suspected that.

BUT, it did lead me to dig into the CSS, and find a couple things that Ian could change to make things visually better in the browser.

**Pull request merged**